socorro/middleware/search_common.py

```diff
@@ -278,7 +278,8 @@ def fix_date_parameter(self, parameters):
                         msg='date must have a prefix operator'
                     )
                 today = datetimeutil.utc_now()
-                if param.value > today:
+                # We add a safety margin to not be too strict.
+                if param.value > today + datetime.timedelta(days=1):
```

Comment:
SuperSearch has always been able to handle missing indices. I believe the reason why we put that in place was because there was a problem when dates where incoherent, like if a date range was exclusive (for example: ``> 2016-10-01 AND < 2016-09-30``).

Wayne's error does not come from Elasticsearch, it comes from our validation process. I believe his clock is off by a few seconds and that triggers the "no future" condition. You don't need to fake your clock to reproduce that. In the form, just enter a ``to`` date that is slightly ahead of the current time.

It is not possible to rely on a server-side date for this. Or rather, it is, but that would be very inconvenient, and would lead to weirdness (if you leave the page open for a day, come back to it, and click the shortcuts to make the date up-to-date, you would get yesterday's datetime). 

I am now convinced that we should remove the "no future" validation step from Super Search, and keep the rest (make sure the date range is coherent). I'm closing this and will work on something different.